hostile/1.1.0/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# Set global variables
+ARG HOSTILE_VER="1.1.0"
+ARG BOWTIE2_VER="2.5.1"
+
+# Stage 1: Build Dockerfile
+FROM ubuntu:focal AS builder
+ARG HOSTILE_VER
+ARG BOWTIE2_VER
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    software-properties-common && \
+    add-apt-repository ppa:deadsnakes/ppa && \
+    apt-get update && apt-get install -y --no-install-recommends \
+    python3.10 python3.10-distutils build-essential \
+    wget unzip samtools minimap2 bedtools gawk ca-certificates && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Install pip for Python 3.10
+RUN wget https://bootstrap.pypa.io/get-pip.py && python3.10 get-pip.py && \
+    rm get-pip.py && pip install --no-cache-dir setuptools pytest && \
+    ln -sf /usr/bin/python3.10 /usr/bin/python3 && \
+    ln -sf /usr/local/bin/pip /usr/bin/pip
+
+# Install bowtie2 and ensure all executables are accessible
+RUN wget https://github.com/BenLangmead/bowtie2/releases/download/v${BOWTIE2_VER}/bowtie2-${BOWTIE2_VER}-linux-x86_64.zip && \
+    unzip bowtie2-${BOWTIE2_VER}-linux-x86_64.zip && \
+    mv bowtie2-${BOWTIE2_VER}-linux-x86_64 /usr/local/bowtie2 && \
+    ln -s /usr/local/bowtie2/bowtie2* /usr/bin/ && \
+    rm -f bowtie2-${BOWTIE2_VER}-linux-x86_64.zip
+
+# Install hostile
+RUN wget https://github.com/bede/hostile/archive/refs/tags/${HOSTILE_VER}.tar.gz && \
+    tar -xzvf ${HOSTILE_VER}.tar.gz && cd hostile-${HOSTILE_VER} && \
+    pip install --no-cache-dir . && \
+    pytest && \
+    cd .. && rm -rf ${HOSTILE_VER}.tar.gz
+
+
+# Stage 2: Final image
+FROM ubuntu:focal AS app
+
+LABEL base.image="ubuntu:focal" 
+LABEL dockerfile.version="1" 
+LABEL software="hostile"
+LABEL software.version="${HOSTILE_VER}"
```

Comment:
Can you add the following to line 41-ish (a line after the app stage starts, but before the LABEL lines).

```
ARG HOSTILE_VER
```
Right now, the `software.version` label isn't registering and is causing a GA action test to fail.

Reply:
yeah must have missed that, fixed now!